kclvm/sema/src/resolver/import.rs

```diff
@@ -39,7 +41,7 @@ impl<'ctx> Resolver<'ctx> {
                                     pos: Position {
                                         filename: m.filename.clone(),
                                         line: stmt.line,
-                                        column: Some(1),
+                                        column: Some(0),
```

Comment:
Why change 1 to 0?

Reply:
The col in the AST position starts from 0, but in the IDE or CMD it starts from 1.
In the function format_diagnostic(), the value of col is added by one before output.
```
fn format_diagnostic(&mut self, diag: &Diagnostic) -> Vec<String> {
    if let Style::LineAndColumn = msg.style {
        if let Some(column) = msg.pos.column {
            let column = column + 1;
```
So here if it is set to `Some(1)`, the final result is the second column of row. This is different from the usual convention.

Reply:
> The col in the AST position starts from 0, but in the IDE or CMD it starts from 1. In the function format_diagnostic(), the value of col is added by one before output.
> 
> ```
> fn format_diagnostic(&mut self, diag: &Diagnostic) -> Vec<String> {
>     if let Style::LineAndColumn = msg.style {
>         if let Some(column) = msg.pos.column {
>             let column = column + 1;
> ```
> 
> So here if it is set to `Some(1)`, the final result is the second column of row. This is different from the usual convention.

OK, I have recorded an issue #104  related to these, u can leave some comments and TODO to explain why it is 0